chart/values.schema.json

```diff
@@ -1685,6 +1685,21 @@
                         }
                     }
                 },
+                "extraContainers": {
+                    "description": "Launch additional containers for the create user job pod",
+                    "type": "array",
+                    "default": []
+                },
+                "extraVolumes": {
+                    "description": "Mount additional volumes into create user job",
+                    "type": "array",
+                    "default": []
+                },
+                "extraVolumeMounts": {
+                    "description": "Mount additional volumes into create user job",
+                    "type": "array",
+                    "default": []
+                },
```

Comment:
```suggestion
                "extraContainers": {
                    "description": "Launch additional containers for the create user job pod",
                    "type": "array",
                    "default": [],
                    "items": {
                        "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.22.0-standalone-strict/container-v1.json"
                    }
                },
                "extraVolumes": {
                    "description": "Mount additional volumes into create user job",
                    "type": "array",
                    "default": [],
                    "items": {
                        "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.22.0-standalone-strict/volume-v1.json"
                    }
                },
                "extraVolumeMounts": {
                    "description": "Mount additional volumes into create user job",
                    "type": "array",
                    "default": [],
                    "items": {
                        "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.22.0-standalone-strict/volumemount-v1.json"
                    }
                },
```

We have more comprehensive validation now.

Reply:
Done!